cmd/mysterium_client/run/command.go

```diff
@@ -98,3 +105,13 @@ func (cmd *CommandRun) Kill() error {
 
 	return nil
 }
+
+func sigtermListener(terminator chan os.Signal, cmd *CommandRun) {
+	<-terminator
+	err := cmd.Kill()
+	if err != nil {
+		fmt.Printf("Unable to kill one of subroutines %q\n", err.Error())
+	}
+	fmt.Println("Good bye")
+	os.Exit(1)
+}
```

Comment:
Closing subroutines and exiting is already implemented in `mysterium.client.go/newStopHandler` - can we re-use that somehow?

Reply:
It's hooked up differently, but the action it does on exit is duplicated:
```go
err := cmd.Kill()
if err != nil {
	fmt.Printf("Unable to kill one of subroutines %q\n", err.Error())
}
fmt.Println("Good bye")
os.Exit(1)
```

```go
if err := cmdRun.Kill(); err != nil {
	return err
}

os.Exit(0)
return nil
```

What bothers me is that these two duplications already differs - one returns exit `1` and prints outs "Good bye", another returns `0` without text.

We can easily DRY these - to move them to a separate method on `CommandRun` or to extend `CommandRun.Kill` with these duplications.